server/utils/bookingSearchUtils.ts

```diff
@@ -63,8 +63,6 @@ export function convertApiStatusToUiStatus(status: BookingSearchApiStatus): Book
   switch (status) {
```

Comment:
Looking into function, do we need BookingSearchUIStatus? Looks like all we need is a function for turning an API Booking Status into a human readable string?

Reply:
Yeah arguably not. Good call, I'll refactor that